go.sum

```diff
@@ -185,8 +187,6 @@ github.com/stretchr/testify v1.2.2/go.mod h1:a8OnRcib4nhh0OaRAV+Yts87kKdq0PP7pXf
 github.com/tarm/serial v0.0.0-20180830185346-98f6abe2eb07/go.mod h1:kDXzergiv9cbyO7IOYJZWg1U88JhDg3PB6klq9Hg2pA=
 github.com/tektoncd/pipeline v0.3.1 h1:LZ0U+HJ3jg3o7e5A7+8nRDEt0G5yJ5seb/uoTxHtBR0=
 github.com/tektoncd/pipeline v0.3.1/go.mod h1:IZzJdiX9EqEMuUcgdnElozdYYRh0/ZRC+NKMLj1K3Yw=
-github.com/tektoncd/plumbing v0.0.0-20190515135757-19ac801cfcf2 h1:F2DzJ7AOA5y6tDBKPL/vwsDgAWQGI67ph+aKThl8rkI=
```

Comment:
why was this deleted? 
Seems like unrelated to this patch.

Reply:
Thanks for your comment! Sorry and you are right. My editor seemed to run `go mod tidy` implicitly. I'll remove the change above and create an another PR to remove those unnecessary dependencies! 👍

Reply:
@sthaha @shuheiktgw also it's on the `go.sum`, ain't a big deal that those are removed :angel: